libs/shared/design-tokens/src/lib/global/typography.yml

```
@@ -14,8 +14,8 @@ props:
     meta:
       friendlyName: Headline 1
       pangram: 'The quick brown fox jumps over a lazy dog'
-      selector: .heading-1
-      mixinName: heading-1
+      selector: .dt-heading-1
```

```suggestion
      selector: .dt-h1
```
What do you think about shortening the selectors a bit - here dt-h1 would work I guess. 
What about `dt-body-up-sem` for `dt-body-uppercased-super-emphasized`?

---

We stated in our Coding guidelines that we don't want to follow abbreviations. So I would avoid it here as well.

---

To be honest in code i totally agree, but those classes need to be learned by our consumers and written by hand repeatedly. I think convenience wins out over not having abbreviations here.